feature/SALOONY-008-otp-page

hazzem-badran opened the pull request.

MohamedEllouze commented on components/atoms/input-field/style.ts:
```diff
@@ -13,6 +13,11 @@ export const styles = StyleSheet.create({
     backgroundColor: '#fff',
   },
+  otpInput: {
+    backgroundColor: '#fff',
```
create the color var

MohamedEllouze commented on components/atoms/input-field/style.ts:
```diff
@@ -13,6 +13,11 @@ export const styles = StyleSheet.create({
   input: {
     backgroundColor: '#fff',
```
color var

MohamedEllouze commented on components/organisms/otp-form/index.tsx:
```diff
@@ -0,0 +1,78 @@
+
+      <View style={styles.descriptionContainer}>
+        <Text variant="bodyMedium" style={styles.descriptionText}>
+          Enter the OTP from SMS that we’ve sent
```
its better to separate the text from the components

MohamedEllouze commented on constants/routes.ts:
```diff
@@ -2,9 +2,11 @@ export const ROUTES = {
 
 export const API_ROUTES = {
   USERS: '/users' as const,
   LOGIN: '/auth/login' as const,
```
it better to replace "/auth" with a variable

MohamedEllouze commented on constants/routes.ts:
```diff
@@ -2,9 +2,11 @@ export const ROUTES = {
 export const API_ROUTES = {
   USERS: '/users' as const,
   LOGIN: '/auth/login' as const,
+  OTP: '/auth/otp' as const,
```
use ROUTES.OTP here

MohamedEllouze commented on views/home/index.tsx:
```diff
@@ -1,41 +1,39 @@
+    //   style={styles.backgroundImage}
+    //   resizeMode="cover"
+    // >
+    //     <View style={styles.container}>
```
don't leave comments!

MohamedEllouze commented on views/login/index.tsx:
```diff
@@ -23,7 +23,7 @@ const LoginPage = () => {
           <ThemedText style={styles.footerText}>Don’t have an account?</ThemedText>
           <Link href={ROUTES.SIGNUP} style={styles.link}>
-            Sign up
+            Sig n up
```
create var for this text

hazzem-badran replied:
رح نعمل الفار لكل النصوص اللي عنا بس ندعم اللغات في التطبيق

hazzem-badran committed:
feature/SALOONY-008-otp-page

# Conflicts:
#	components/organisms/login-form/index.tsx
#	constants/formFields.constants.ts
#	constants/routes.ts

hazzem-badran committed:
Merge branch 'develop' into feature/SALOONY-008-otp-page
